commands/pm/backdrop_pm_enable.drush.inc

```diff
@@ -0,0 +1,147 @@
+<?php
+/**
+ * @file
+ * Drush project management enable command.
+ */
+
+use Drupal\Core\Logger\RfcLogLevel;
+
+/**
+ * Implements hook_drush_command().
+ */
+function backdrop_pm_enable_drush_command() {
+  $items = array();
+  $items['backdrop-pm-enable'] = array(
+    'description' => 'Enable backdrop modules.',
+    'callback' => 'backdrop_command_pm_enable',
+    'hidden' => TRUE,
+    'arguments' => array(
+      'module-name' => array('The name of the module(s) you would like to enable.'),
+    ),
+    'required-arguments' => TRUE,
+    'aliases' => array('en',),
+    'bootstrap' => \Drush\Boot\BackdropBoot::BOOTSTRAP_FULL,
+  );
+
+  return $items;
+}
+
+/**
+ * Command callback enable modules.
+ *
+ * @see _enable_project().
+ */
+function backdrop_command_pm_enable() {
+  $projects = func_get_args();
+
+  // Get modules present in files system that are possible to enable.
+  $module_list = system_rebuild_module_data();
+
+  if (!isset($projects)) {
+    drush_print_r(dt("\tPlease provide module name(s)\n\n"));
+    return;
+  }
+
+  $clear_cache = FALSE;
+  foreach($projects as $project) {
+    // Check if requested module depends on other modules.
+    $dependencies = $module_list[$project]->info['dependencies'];
+    $dependents = [];
+    foreach ($dependencies as $dependency) {
+      if (module_exists($dependency) || in_array($dependency, $projects)) {
+        // Dependency is already accounted for.
+      }
+      else {
+        array_unshift($projects, $dependency);
+        $dependents = array_merge($projects, $dependents);
```

Comment:
something wrong happen with these 2 lines, some projects and/or dependencies are added twice in the `$projects` array, for example when trying to enable the splashify module on a project I got the following message :  `The following projects will be enabled: mobile_detect, jstorage, splashify, jstorage, splashify.`

Reply:
addressed in https://github.com/backdrop-contrib/drush/pull/127